README.md

```diff
@@ -1414,21 +1414,15 @@ meanings we’ll do our best to spell out.
 
 ### Kinds of income
 
-🄳 You must pay federal, state, and in some cases, local **income taxes** on your **income**,
-which is the money you make.
-
-🄳 From a tax perspective, your income is divided between
-[**ordinary income**](https://www.investopedia.com/terms/o/ordinaryincome.asp), which includes
-wages and salary and interest, and
-[**capital gains**](https://www.investopedia.com/terms/c/capital_gains_tax.asp), the term for
-profit you receive from selling assets, including stocks.
-One key difference between ordinary income and capital gains is you can sell assets for a
-profit or a loss, so this kind of income may be taxed differently, with the ability to
-track both gains and losses.
-
-🄳 Capital gains are further divided between **long-term** and **short-term** capital gains.
-Long-term capital gains are what you make from selling assets, such as stock, a business,
-a house, or land, that you held for more than a year.
+🄳 **Income** is the money an individual makes. For tax purposes, there are two main types of income which are taxed differently. [**Ordinary income**](https://www.investopedia.com/terms/o/ordinaryincome.asp) includes
```

Comment:
Comma before "which are taxed"

Reply:
I wavered on this one when I did it. Comma should be in. Fixed.